DiscordBot/Commands/EventCommands.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Threading.Tasks;
+using DSharpPlus.CommandsNext;
+using DSharpPlus.CommandsNext.Attributes;
+using DSharpPlus.Interactivity;
+
+namespace DiscordBot.Commands
+{
+    internal class EventCommands
+    {
+        [Command("event")]
+        public async Task Event(CommandContext context)
+        {
+            await context.RespondAsync(
+                $"{context.Member.Mention} - choose one of the actions below or answer ``stop`` to cancel. (time out in 30s)\n" +
+                "``create`` - create new event.\n" +
+                "``rm`` - delete event.\n" +
+                "``edit`` - edit event.\n"
+            );
+
+            var interactivity = context.Client.GetInteractivityModule();
+            var response = await interactivity.WaitForMessageAsync(
+                message =>
+                    message.Content.ToLower() == "create" ||
+                    message.Content.ToLower() == "rm" ||
+                    message.Content.ToLower() == "edit" ||
+                    message.Content.ToLower() == "stop",
```

Comment:
I would extract this to a small function so that you can create a variable for the lowercase value. I'd probably also store the strings as private constants 🙂

Reply:
Should this be in a helper command?

Reply:
I think just a private method here is fine 🙂